tests/unit/dataactvalidator/test_scripts_loadTas.py

```diff
@@ -59,12 +59,14 @@ def test_cleanTas_space_nulls(tmpdir):
 
 
 def test_updateTASLookups(database, monkeypatch):
```

Comment:
This test is great. My only overall suggestion might be to make the setup a bit more explicit for the benefit of people who might be familiar with how the TAS load is supposed to work but less familiar with the constructs of unit testing (people like me _cough cough_).

For example, does it make sense to do something like:

```python
incoming_tas_data = pd.DataFrame({
'availability_type_code': ['0', '1', '2'],
'account_num': [111, 222, 333],
'agency_identifier': ['new', '1', 'new'],
etc.
})

existing_tas_data = the list of TAS factories from line 80

# add existing tas to db
sess.add_all(the list of TAS factories)

# monkeypatch the TAS load to return the mocked incoming .csv values
monkeypath.setattr(loadTas, 'cleanTas, Mock(etc etc)
```

Feel free to ignore this suggestion if it's too rudimentary, but I think it easier to think of these tests in very basic terms like "here is the new stuff" and "here is the stuff that's already in the tas table"